packages/base/src/dialogs/symbology/components/color_ramp/ColorRampValueControls.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from '@jupyterlab/ui-components';
+import React from 'react';
+
+import { IColorRampValueControlsProps } from '@/src/types';
+
+export const ColorRampValueControls: React.FC<IColorRampValueControlsProps> = ({
+  min,
+  setMin,
+  max,
+  setMax,
+  rampDef,
+  initialMin,
+  initialMax,
```

Comment:
Maybe instead of `initialMin` and `initialMax`, we could use `dataMin` and `dataMax`? And instead of `min`/`max`, `selectedMin`/`selectedMax`?